orchagent/portsorch.cpp

```diff
@@ -1502,6 +1504,37 @@ void PortsOrch::initializeQueues(Port &port)
                 to_string(queueIndex));
         queueIndexVector.push_back(queueIndexTuple);
 
+        sai_attribute_t attr;
+        attr.id = SAI_QUEUE_ATTR_TYPE;
+
+        sai_status_t status = sai_queue_api->get_queue_attribute(port.m_queue_ids[queueIndex], 1, &attr);
+        if (status == SAI_STATUS_SUCCESS)
+        {
+            string queueType;
+
+            switch (attr.value.s32)
+            {
+            case SAI_QUEUE_TYPE_ALL:
+                queueType = "SAI_QUEUE_TYPE_ALL";
+                break;
+            case SAI_QUEUE_TYPE_UNICAST:
+                queueType = "SAI_QUEUE_TYPE_UNICAST";
+                break;
+            case SAI_QUEUE_TYPE_MULTICAST:
+                queueType = "SAI_QUEUE_TYPE_MULTICAST";
+                break;
+            default:
+                throw runtime_error("Unsupported queue " + to_string(queueIndex) +
```

Comment:
>runtime_error [](start = 22, length = 13)

We prefer short fixed string in exception string. You may use SWSS_LOG_ERROR before it.